components/text/text-utilities/Makefile

```diff
@@ -90,7 +83,7 @@ COMPONENT_TEST_ENV +=	TS_CMD_HEXDUMP=$(PROTOUSRBINDIR)/hexdump
 COMPONENT_TEST_ENV +=   TS_CMD_RENAME=$(PROTOUSRBINDIR)/rename
 COMPONENT_TEST_ENV +=	PATH="$(PATH.gnu)"
 # Needed for the libsmartcols.so library
-COMPONENT_TEST_ENV +=	LD_LIBRARY_PATH=$(PROTOUSRLIBDIR64)
+#COMPONENT_TEST_ENV +=	LD_LIBRARY_PATH=$(PROTOUSRLIBDIR64)
```

Comment:
What is this?

The comment says: **Needed for the libsmartcols.so library**

So, is the line needed or not?  If it _is_ needed then why is it commented out?  If it _is not_ needed then why the comment says it is needed?